packages/onchainkit/scripts/get-next-version.js

```diff
@@ -2,37 +2,59 @@ import { execSync } from 'child_process';
 import process from 'process';
 import fs from 'fs';
 import path from 'path';
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
```

Comment:
nit: can put this named import with the default import on line 4

Reply:
Removed this and using `path.dirname` below for consistency.